docs/howtos/resource-override.md

```diff
@@ -186,6 +231,39 @@ The `jsonPatchOverrides` field supports the following fields:
 
 - `value`: The value to be set.
    * If the `op` is `remove`, the value cannot be set.
+  * There is a list of reserved variables that will be replaced by the actual values:
+      * `${MEMBER-CLUSTER-NAME}`:  this will be replaced by the name of the `memberCluster` that represents this cluster.
+
+For example, to add a label to the `Deployment` named `my-deployment` on clusters with the label `env: prod`,
+you can use the following configuration:
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1alpha1
+kind: ResourceOverride
+metadata:
+  name: example-ro
+  namespace: test-namespace
+spec:
+  placement:
+    name: crp-example
+  resourceSelectors:
+    -  group: apps
+       kind: Deployment
+       version: v1
+       name: my-deployment
+  policy:
+    overrideRules:
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: prod
+        jsonPatchOverrides:

```

Comment:
Can we also have an example for delete? thanks!

Reply:
it's in line 165